sql-parser/src/main/java/io/crate/sql/parser/Statement.g

```diff
@@ -1280,6 +1300,7 @@ KEY: 'KEY';
 DUPLICATE: 'DUPLICATE';
 SET: 'SET';
 RESET: 'RESET';
+DEFAULT: 'DEFAULT';
```

Comment:
I would accept a breaking change (of course needs to be documented) 
`DEFAULT` is reserved in postgresql, mssql, sql:2003, sql:1999 and sql-92
https://www.postgresql.org/docs/8.3/static/sql-keywords-appendix.html